offline.appcache

```diff
@@ -0,0 +1,44 @@
+CACHE MANIFEST
+# 2016-01-14 v1.1.3
+
```

Comment:
Probably a good idea to add the `CACHE:` section here in case we want to add `NETWORK:` and `FALLBACK:` section later?